src/fw.rs

```diff
@@ -202,7 +204,7 @@ impl Output {
             };
             writeln!(
                 file,
-                "{}\t{}{}\t{}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}",
+                "{}\t{}{}\t{}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}\t{:.3}",
```

Comment:
Those `writeln!`s are becoming very unwieldy and hard to track - might refactor that as nowadays you can interpolate variables straight into format strings (not sure you could do that when I first wrote this).